### PR TITLE
Development to master

### PR DESCRIPTION
**Pull request by Dongmo12**
Everything should be good now. Thanks for your help!!!

### CODE REVIEW

**Comment by AdnanAfsari** [comment]
Hello @Dongmo12 :) 

Previous problem solved but arises a new one but do not worry its not a big one.

look at your down section- it is not taking the entire spaces but it should and also be equally distributed as the original one.
Screenshot 2019-05-18 at 2 31 20 AM



and looks at the original one 
Screenshot 2019-05-18 at 2 33 39 AM

Please fix this issue and submit for another review :).Stay strong, i know the feeling but you have to do it.

Happy Coding

**Comment by Torres-ssf** [comment]
### Great work team 👏
You did a nice job on this project. But I have to ask, this project was supposed to be done individually right? What happened?
We need to make some changes in this project to make it in order with Microverse guidelines. The changes are:

- [x] The main content should be wrapped around `main` tags and have an image set as background

- [x] The navbar should be inside the main tag

- The 3 main sentences:
- [x] There should be three links
- [x] Each link is an `a` HTML tag, with a `div` tag inside it.
- [x] The `div` tag inside the `a` tag should use an image as background. The text on the original page is actually an image

- [x] Your last sentence is a lit bit under the next section. If possible try to match the original design. you can message me on slack for help if you need.

- Footer

- [x] In this page the footer must starts at the div class="secondary-content". Its divides in 2 parts, top, and bottom.

- [x] All 4 images must be inside an `a` tag each. And inside each `a` tag must be a div tag. You are going to load the images using only the background from the div tag. No need for img tags here.

- [x] It seems that the bottom part of your footer it's on top the upper part.

- [x]  There is one error in your HTML code. Check them using this HTML [validator](https://validator.w3.org/) . Ignore the warnings.

Kindly make these changes and submit for code review again.

**Comment by bolah2009** [comment]
PR currently been reviewed

File: Assets/stylesheet.css [hunk 1]
```diff
@@ -0,0 +1,131 @@
+ /* *{
+    -webkit-box-sizing: border-box;
+    -moz-box-sizing: border-box;
+    box-sizing: border-box;
+} */
```
Comment (bolah2009) [inline Assets/stylesheet.css]: Consider deleting commented out code.

File: Assets/stylesheet.css [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+    box-shadow: 3px 1px #3c3c3c;
+    background: inherit;
+}
+main{    
+    position: relative;
+    background-image: url(./main_background.jpg);
+    background-position: 50% 100%;
+    background-repeat: no-repeat;
+    height: 538px;
+    width: 1268px;
```
Comment (bolah2009) [inline Assets/stylesheet.css]: `SUGGESTION`

Let's reposition the background image to fill the page as shown below:

|Current|Preffered|
| :-----: | :------: |
| appl com current  |  preffered app |

File: index.html [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+        <link rel="stylesheet" href="./Assets/stylesheet.css">
+    </head>
+    <body>
+        
+            
+        
+        <main>
+                <nav>
+                    <ul>
+                        <li><a href="#"><i class="fab fa-apple"></i></a></li>
```
Comment (bolah2009) [inline index.html]: `REQUIRED`

The home icon (Apple image) seems not to be showing on the page. See below:


|Current|Preffered|
| :-----: | :------: |
| current apple home icon  | preffered apple home icon png  |

File: Assets/stylesheet.css [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+    padding: 0;
+    margin-top: 0;
+}
+nav a{
+    text-shadow: 2px 2px #363636;
+    text-decoration: none;
+    color: white;
+}
+nav li{
+    display: flex;
```
Comment (bolah2009) [inline Assets/stylesheet.css]: `REQUIRED`

Let's add `hover` effects to the `li` elements of the navbar. See image below:

|Current|Preffered|
| :-----: | :------: |
| current apple nav hover  | preffered apple nav hover  |

File: index.html [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+                        <li><a href="#">Store</a></li>
+                        <li><a href="#">Mac</a></li>
+                        <li><a href="#">iPod</a></li>
+                        <li><a href="#">iPhone</a></li>
+                        <li><a href="#">iPad</a></li>
+                        <li><a href="#">iTunes</a></li>
+                        <li><a href="#">Support</a></li>
+                        <li class="search"><form action="Search">
+                            <i class="fas fa-search"></i><input type="search" aria-label="search">
```
Comment (bolah2009) [inline index.html:29]: `SUGGESTION`

The `fas fa-search` is not rendering, it may be that you didn't link the `font awesome` assets properly, :thinking: ?

**Review by bolah2009 — CHANGES_REQUESTED** [review]
## Nice Job [Armand](../../) :clap:

Awesome job so far, the [clone](https://raw.githack.com/Dongmo12/Building-with-Backgrounds-and-Gradients/development/index.html) looks nice. :clap: :+1: 

#### Just a few adjustment as explained in the comments below.

Good luck with your next submission.

Cheers
[Bola](https://bolabuari.com/)

**Comment by bolah2009** [comment]
Currently under review

**Review by bolah2009 — APPROVED** [review]
## Awesome Job [Armand](../../) :clap:

### Some suggested changes are not yet implemented (check comment [1](https://github.com/Dongmo12/Building-with-Backgrounds-and-Gradients/pull/4#discussion_r308551772), [2](https://github.com/Dongmo12/Building-with-Backgrounds-and-Gradients/pull/4#discussion_r308553675), [3](https://github.com/Dongmo12/Building-with-Backgrounds-and-Gradients/pull/4#discussion_r308555748) and [4](https://github.com/Dongmo12/Building-with-Backgrounds-and-Gradients/pull/4#discussion_r308556670) ), I strongly recommend making those changes before merging.

Good luck with your next project.

Cheers
[Bola](https://bolabuari.com/)

**Comment by bolah2009** [comment]
# PR is approved :heavy_check_mark: :+1:

**Pull request merged**